conda_package/mpas_tools/mesh/creation/sort_mesh.py

```diff
@@ -0,0 +1,187 @@
+        sort_rev(mesh["cellsOnEdge"], cell_rev)
+    mesh["cellsOnVertex"][:] = \
+        sort_rev(mesh["cellsOnVertex"], cell_rev)
+
+    for var in mesh.keys():
+        dims = mesh.variables[var].dims
+        if ("nCells" in dims):
+            mesh[var][:] = sort_fwd(mesh[var], cell_fwd)
+
+    mesh["indexToCellID"][:] = np.arange(ncells) + 1
+
+    # sort duals via pseudo-linear cell-wise ordering
+
+    dual_fwd = np.ravel(mesh["verticesOnCell"].values)
+    dual_fwd = dual_fwd[dual_fwd > 0]
+
+    __, imap = np.unique(dual_fwd, return_index=True)
+
+    dual_fwd = dual_fwd[np.sort(imap)]
+
+    dual_rev = np.zeros(nduals, dtype=np.int32)
+    dual_rev[dual_fwd - 1] = np.arange(nduals) + 1
+
+    mesh["verticesOnCell"][:] = \
+        sort_rev(mesh["verticesOnCell"], dual_rev)
+
+    mesh["verticesOnEdge"][:] = \
+        sort_rev(mesh["verticesOnEdge"], dual_rev)
+
+    for var in mesh.keys():
+        dims = mesh.variables[var].dims
+        if ("nVertices" in dims):
+            mesh[var][:] = sort_fwd(mesh[var], dual_fwd)
+
+    mesh["indexToVertexID"][:] = np.arange(nduals) + 1
+
+    # sort edges via pseudo-linear cell-wise ordering
+
+    edge_fwd = np.ravel(mesh["edgesOnCell"].values)
+    edge_fwd = edge_fwd[edge_fwd > 0]
+
+    __, imap = np.unique(edge_fwd, return_index=True)
+
+    edge_fwd = edge_fwd[np.sort(imap)]
+
+    edge_rev = np.zeros(nedges, dtype=np.int32)
+    edge_rev[edge_fwd - 1] = np.arange(nedges) + 1
+
+    mesh["edgesOnCell"][:] = \
+        sort_rev(mesh["edgesOnCell"], edge_rev)
+
+    mesh["edgesOnEdge"][:] = \
+        sort_rev(mesh["edgesOnEdge"], edge_rev)
+
+    mesh["edgesOnVertex"][:] = \
+        sort_rev(mesh["edgesOnVertex"], edge_rev)
+
+    for var in mesh.keys():
+        dims = mesh.variables[var].dims
+        if ("nEdges" in dims):
+            mesh[var][:] = sort_fwd(mesh[var], edge_fwd)
+
+    mesh["indexToEdgeID"][:] = np.arange(nedges) + 1
+
+    return mesh
+
+
+def cell_del2(mesh):
+    """
+    CELL-DEL2: form cell-to-cell sparse adjacency graph
+
+    """
+    xvec = np.array([], dtype=np.int8)
+    ivec = np.array([], dtype=np.int32)
+    jvec = np.array([], dtype=np.int32)
+
+    topolOnCell = mesh["nEdgesOnCell"].values
+    cellsOnCell = mesh["cellsOnCell"].values
+
+    for edge in range(np.max(topolOnCell)):
+
+        # cell-to-cell pairs, if edges exist
+        mask = topolOnCell > edge
+        idx_self = np.argwhere(mask).ravel()
+        idx_next = cellsOnCell[mask, edge] - 1
+
+        # cell-to-cell pairs, if cells exist
+        mask = idx_next >= 0
+        idx_self = idx_self[mask]
+        idx_next = idx_next[mask]
+
+        # dummy matrix values, just topol. needed
+        val_edge = np.ones(idx_next.size, dtype=np.int8)
+
+        ivec = np.hstack((ivec, idx_self))
+        jvec = np.hstack((jvec, idx_next))
+        xvec = np.hstack((xvec, val_edge))
+
+    return csr_matrix((xvec, (ivec, jvec)))
+
+
+if (__name__ == "__main__"):
```

Comment:
To function as an entry point, this needs to be a function:
```suggestion
def main():
```
Then, at the bottom of the file, you can have:
``` python
if (__name__ == "__main__"):\
    main()
```